lib/app/models/module.rb

```diff
@@ -4,18 +4,20 @@
 
 require_relative 'module/test_group'
 require_relative 'module/test_set'
+require_relative 'module/tag'
 
 module Inferno
   class Module
     @modules = {}
 
-    attr_accessor :default_test_set
+    attr_accessor :name
```

Comment:
It would be nice to keep these in alphabetical order.

Reply:
Ah, I didn't realize that was the reasoning behind the order.  I reordered to match the assignments that occurred in initialize, but I'm good with doing alphabetical here.  Fixed.